_apps/todo.md

```diff
@@ -0,0 +1,43 @@
+---
+title: todo
+description: Creates new issues from actionable comments in your code.
+slug: todo
+screenshots:
+- https://user-images.githubusercontent.com/10660468/31048765-83569c30-a5f2-11e7-933a-a119d43ad029.png
+authors: [JasonEtco]
+repository: JasonEtco/todo
+stars: 10
+updated: 2017-09-30 05:57:19 UTC
```

Comment:
`stars` and `updated` are automatically synced so no need to list them here.

Reply:
When do they get synced? At deploy time? I had built it locally and it didn't show anything :(